### DISCUSSION TITLE
German Translation is done

### DISCUSSION THREAD

**Discussion by WebCrew** [givanz/Vvveb#35] [General]
Hi **Givan** and **Community**,

I finished the German translation. I made an issue about it. You can find it here:
https://github.com/givanz/VvvebJs/issues/310#issuecomment-1666541173

The **de_DE** folder include a **landing.mo** and a **landing.po** file.

There is no need to mention me at the README.md or so but You can if You want to. I did it from heart and just to support that awesome project.

Kind regards, Andreas, GitHub @WebCrew 👍🏼

**Comment by givanz** [comment]
Hi Andreas,

Thank you for your contribution and support.
The translations are committed to the repository and included in the latest.zip.
I included your user in the README and gave permissions to the repository.

**Comment by ALNIuser** [comment]
How do I install and enable it? Which folder on the server do you want to download to?

**Comment by givanz** [comment]
The language pack is bundled with the zip and is located in `/locale/de_DE/`

You only need to set the language to active from `admin > settings > localization > languages`

![language-activate](https://github.com/givanz/Vvveb/assets/302738/2aecd42d-5e8d-40ad-85bc-ba1d23cbaef0)

To install a new language click install language in the language configuration page

![language-install](https://github.com/givanz/Vvveb/assets/302738/bd70b1db-44f9-4611-96c0-c2adb5f98a6f)

The files are downloaded from https://github.com/Vvveb/ru_RU and copied to  `/locale/ru_RU/`
